pages/packages/index.tsx

```diff
@@ -1,11 +1,93 @@
-import DocTemplate from '../docs/[...slug]'
-import { getDocProps } from '../../utils/docs/getDocProps'
+import { DocsGrid, DocGridContent } from '../docs/[...slug]'
 import { GetStaticProps } from 'next'
 import { GithubError } from 'next-tinacms-github'
+import React, { useState } from 'react'
+import { createTocListener } from 'utils'
+import { NextSeo } from 'next-seo'
+import { openGraphImage } from 'utils/open-graph-image'
+import { DocsLayout, MarkdownContent } from 'components/layout'
+import { getJsonPreviewProps, readJsonFile } from 'utils/getJsonPreviewProps'
+import path from 'path'
+
+
+export default function PackageIndex(props) {
+  const excerpt = 'Packages for TinaCMS.'
+
+  const name = "Packages"
+
+  const contentRef = React.useRef<HTMLDivElement>(null)
+  const isBrowser = typeof window !== `undefined`
+  const [activeIds, setActiveIds] = useState([])
+
+  React.useEffect(() => {
```

Comment:
There isn't a TOC in the template so we should remove these event listeners if were not going to add that.